media/transcoder/create_job_with_concatenated_inputs.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_with_concatenated_inputs]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1"
+)
+
+// createJobWithConcatenatedInputs creates a job that concatenates two input
+// videos. See https://cloud.google.com/transcoder/docs/how-to/concatenate-videos
+// for more information.
+func createJobWithConcatenatedInputs(w io.Writer, projectID string, location string, input1URI string, startTimeInput1 time.Duration, endTimeInput1 time.Duration, input2URI string, startTimeInput2 time.Duration, endTimeInput2 time.Duration, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// input1URI := "gs://my-bucket/my-video-file1"
+	// startTimeInput1 := 0*time.Second
+	// endTimeInput1 := 8*time.Second + 100*time.Millisecond
+	// input2URI := "gs://my-bucket/my-video-file2"
+	// startTimeInput2 := 3*time.Second + 500*time.Millisecond
+	// endTimeInput2 := 15*time.Second
+	// outputURI := "gs://my-bucket/my-output-folder/"
+
+	ctx := context.Background()
+	client, err := transcoder.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &transcoderpb.CreateJobRequest{
```

Comment:
Agreed. Unfortunately, most of these JobConfigs are gnarly in this way (see https://github.com/GoogleCloudPlatform/golang-samples/blob/main/media/transcoder/create_job_with_periodic_images_spritesheet.go).

I've tried breaking these into separate variables before and it really didn't help things since you had to skip around in the file to find the definitions for everything.